cfme/automate/buttons.py

```diff
@@ -2,21 +2,32 @@
 import attr
 import re
 
-from navmazing import NavigateToSibling, NavigateToAttribute
-
+from navmazing import NavigateToAttribute, NavigateToSibling
 from widgetastic.utils import ParametrizedString, VersionPick
+from widgetastic.widget import (
+    Checkbox,
+    ColourInput,
+    ConditionalSwitchableView,
+    ParametrizedView,
+    Text,
+    View,
+)
 from widgetastic.xpath import quote
-from widgetastic.widget import Checkbox, ConditionalSwitchableView, ParametrizedView, Text, View
-from widgetastic_manageiq import (SummaryFormItem, FonticonPicker, PotentiallyInvisibleTab,
-                                  RadioGroup, FakeWidget)
 from widgetastic_patternfly import BootstrapSelect, Button, CandidateNotFound, Input
 
 from cfme.modeling.base import BaseCollection, BaseEntity
-from cfme.utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
 from cfme.utils.blockers import BZ
 from cfme.utils.update import Updateable
 from cfme.utils.version import Version
 from cfme.utils.wait import wait_for
+from widgetastic_manageiq import (
+    FakeWidget,
+    FonticonPicker,
+    PotentiallyInvisibleTab,
+    RadioGroup,
+    SummaryFormItem,
+)
```

Comment:
**REQUIRED**
This import should be in the third-party section.

Reply:
@quarckster  `wt_manageiq` we treat as internal right not like `wt.core` and `wt.pfly`

Reply:
@mshriver is something changed? I was thinking that eventually widgetastic_manageiq will go to a separate repo.

Reply:
It can/should eventually be separated - but until then its a local import and should be treated as such, imported in the 3rd group.